tests/test_client_v2_integration.py

```diff
@@ -65,20 +70,26 @@ def test_parse_file_filled_single_page_must_succeed(
     Upload a filled single-page JPEG and verify that common fields are present.
     """
     input_path: Path = PRODUCT_DATA_DIR / "financial_document" / "default_sample.jpg"
-    assert input_path.exists(), f"sample file missing: {input_path}"
 
-    input_doc = PathInput(input_path)
-    options = InferenceParameters(findoc_model_id)
+    input_source = PathInput(input_path)
+    params = InferenceParameters(
+        model_id=findoc_model_id,
+        rag=False,
+        raw_text=False,
+        polygon=False,
+        confidence=False,
+    )
```

Comment:
```suggestion
    params = InferenceParameters(
        model_id=findoc_model_id,
        rag=False,
        raw_text=False,
        polygon=False,
        confidence=False,
        webhook_ids=[],
        alias='py_integration_empty_multipage'
    )
```

Might as well, right?